_data/projects/quickperf-examples.yml

```
@@ -0,0 +1,17 @@
+---
+name: QuickPerf examples
+desc: >-
+  Examples showing how to use [QuickPerf](https://github.com/quick-perf/quickperf) together with various frameworks (JUnit 4, JUnit 5, TestNG, Spring Boot, Micronaut, ...)
+site: https://github.com/quick-perf/quickperf-examples
+tags:
+  - java
+  - performance
+  - quickperf
+  - springboot
+  - micronaut
+  - junit4
+  - junit5
+  - testng
+upforgrabs:
+  name: ":thumbsup: good first issue"
```

This needs to be the label name itself:

```suggestion
  name: "good first issue"
```